pages/book/operators.mdx

```diff
@@ -0,0 +1,524 @@
+
+Binary logical OR ([logical disjunction](https://en.wikipedia.org/wiki/Logical_disjunction)) operator `||{:tact}` returns `false{:tact}` only if both operands are `false{:tact}`, and `true{:tact}` otherwise. It's short-circuited, meaning that it would immediately evaluate the whole expression as `true{:tact}` if the left operand is `true{:tact}`, without evaluating the right one.
+
+Can only be applied to values of type [`Bool{:tact}`](/book/types#booleans):
+
+```tact
+let iLikeSnails: Bool = false;
+iLikeSnails && true;  // true
+iLikeSnails && false; // false, evaluated both operands
+true && iLikeSnails;  // true, didn't evaluate iLikeSnails
+```
+
+## Ternary, `?:` [#ternary]
+
+Conditional (_ternary_) operator is the only Tact operator that takes three operands: a condition followed by a question mark (`?{:tact}`), then an expression to execute if the condition is evaluated to `true{:tact}` followed by a colon (`:{:tact}`), and finally the expression to execute if the condition is evaluated to `false{:tact}`. This operator is frequently used as an alternative to an [`if...else{:tact}`](/book/statements#if-else) statement.
+
+Condition must resolve to type [`Bool{:tact}`](/book/types#booleans):
+
+```tact
+// condition
+// ↓
+true ? "incredibly so" : "absolutely not"; // "incredibly so"
+//     ---------------   ----------------
+//     ↑                 ↑
+//     |                 alternative, when condition is false
+//     |
+//     consequence, when condition is true
+
+2 + 2 == 4 ? true : false; // true
+```
+
+Ternary operator is the only operator with right associativity, besides [assignment-related ones](#assignment). This means that in ambiguous situations Tact would prefer the longest matching sequence. In short, this makes bracket-less nesting of ternary operators possible, but only for alternative cases (the part that comes after the colon sign `:{:tact}`):
+
+```tact
+// don't need additional parentheses for alternative cases
+false ? 1 : (false ? 2 : 3); // 3
+false ? 1 : false ? 2 : 3;   // also 3
+false ? 1 : true ? 2 : 3;    // 2
+
+// need additional parentheses for consequence cases (parts in-between ? and :)
+false ? (false ? 1 : 2) : 3; // 3
+false ? false ? 1 : 2 : 3;   // SYNTAX ERROR!
+true ? (false ? 1 : 2) : 3;  // 2
+```
+
+## Assignment, `=` [#assignment]
+
+Assignment operator `={:tact}` is used to assign a value to a variable, or to a property of a [Message](/book/composite-types#messages) or a [Struct](/book/composite-types#structs). The assignent is a statement and it doesn't return a value.
+
+```tact
+let someVar: Int = 5;    // assignment operator = is used here...
+someVar = 4;             // ...and here
+someVar = (someVar = 5); // SYNTAX ERROR!
+```
+
+### Augmented assignment
+
+Augmented (or compound) assignment operators such as `+={:tact}` combine an operation with an [assignment](#assignment). . The augmented assignment is a statement and it doesn't return a value.
```

Comment:
We can add that, for instance, the statement `x += a` is semantically equivalent to `x = x + a`